src/module.c

```diff
@@ -4380,6 +4380,12 @@ void moduleLoadFromQueue(void) {
                 loadmod->path);
             exit(1);
         }
+        sdsfree(loadmod->path);
+        for (int i = 0; i < loadmod->argc; i++) {
+            decrRefCount(loadmod->argv[i]);
+        }
+        zfree(loadmod);
```

Comment:
@soloestoy I think here is potential memory leak.
in queueLoadModule
```
loadmod->argv = zmalloc(sizeof(robj*)*argc);
```

so we need to below code before zfree(loadmod);
```
zfree(loadmod->argv);
```

Reply:
Yes, I forgot `argv` 😅 , thanks for your remind